packages/app/src/components/Randomizer/Randomizer.test.js

```diff
@@ -77,6 +77,19 @@ test('pressing the button calls the restaurant service', async () => {
   expect(restaurantService.getAllMatches).toBeCalled()
 })
 
+test('pressing the button calls the restaurant service to increase resultAmount', async () => {
+  const { queryByTestId } = await actRender(<Randomizer />)
```

Comment:
Use the `<TestRandomizer />` component alias instead for more sensible default (in test) values for the roll times.
```suggestion
  const { queryByTestId } = await actRender(<TestRandomizer />)
```